packages/widget/src/pages/SwapExecutionPage/useBroadcastedTxs.ts

```diff
@@ -51,19 +51,19 @@ export const useBroadcastedTxsStatus = ({
         );
       });
 
-      const _isAllTxSuccess = transferEvents.every((tx) => {
-        return tx.status === "completed";
-      });
-
```

Comment:
`_isAllTxSuccess` is the root of the issue, sometimes transferEvents all succeed even though the overall swap fails